example/spec/scaling/activityScalingSpec.js

```diff
@@ -0,0 +1,131 @@
+let alarmEvaluationPeriods;
+let alarmPeriodSeconds;
+let sleepMs;
+let clusterArn;
+let numActivityTasks;
+let minInstancesCount;
+const workflowName = 'HelloWorldActivityWorkflow';
+
+describe('scaling for step function activities', () => {
+  beforeAll(async () => {
+    cloudformationResources = (await loadCloudformationTemplate(config)).Resources;
+    activitiesWaitingAlarm = cloudformationResources.ActivitiesWaitingAlarm;
+    alarmEvaluationPeriods = activitiesWaitingAlarm.Properties.EvaluationPeriods;
+    const alarmPeriod = activitiesWaitingAlarm.Properties.Metrics[1].MetricStat.Period;
+    alarmPeriodSeconds = alarmPeriod / alarmEvaluationPeriods;
+    sleepMs = 2 * alarmPeriodSeconds * 1000;
+    clusterArn = await getClusterArn(stackName);
+    numActivityTasks = Object.values(cloudformationResources).filter((resource) => resource.Type === 'AWS::StepFunctions::Activity').length;
+    minInstancesCount = cloudformationResources.CumulusECSAutoScalingGroup.UpdatePolicy.AutoScalingRollingUpdate.MinInstancesInService;
+  });
+
+  it('cloudformation stack has an alarm for ActivitiesWaiting ', () => {
+    expect(activitiesWaitingAlarm.Type).toEqual('AWS::CloudWatch::Alarm');
+  });
+
+  it('HelloWorld ECS Service is a scalable target', () => {
+    const helloWorldScalableTarget = cloudformationResources.HelloWorldServiceECSServiceScalableTarget;
+    expect(helloWorldScalableTarget.Type).toEqual('AWS::ApplicationAutoScaling::ScalableTarget');
+  });
+
+  it('ActivitiesWaitingAlarm is configured to scale the ECSService', () => {
+    const alarmAction = activitiesWaitingAlarm.Properties.AlarmActions[0].Ref;
+    expect(alarmAction).toEqual('ScaleOutTasksECSServiceApplicationScalingPolicy');
+  });
+
+  describe('scaling the service\'s desired tasks', () => {
+    let workflowExecutionArns = [];
+    numExecutions = 10;
+
+    beforeAll(async () => {
+      const workflowExecutionPromises = [];
+
+      for (let i = 0; i < numExecutions; i += 1) {
+        workflowExecutionPromises.push(buildAndStartWorkflow(
+          stackName,
+          config.bucket,
+          workflowName,
+          null,
+          null,
+          {
+            sleep: sleepMs
+          }
+        ));
+      }
+      workflowExecutionArns = await Promise.all(workflowExecutionPromises);
+    });
+
+    describe('when activities waiting are greater than the threshold', () => {
+      it('the number of tasks the service is running should increase', async () => {
+        // wait the period of the alarm plus a bit
+        await sleep(alarmPeriodSeconds * 1000 + 30000);
+        const clusterStats = await getClusterStats(stackName);
+        console.log(`clusterStats ${JSON.stringify(clusterStats, null, 2)}\n`);
+        const runningEC2TasksCount = parseInt(find(clusterStats, ['name', 'runningEC2TasksCount']).value, 10);
+        expect(runningEC2TasksCount).toBeGreaterThan(numActivityTasks);
+      });
+
+      it('adds new ec2 resources', async () => {
```

Comment:
I really like this verbiage in tests and wish we did it more.  I thought the team had moved more toward AVA.  Is that the case?  Also, not a huge deal, but there are a couple of other "it" calls that don't read as cleanly, like the prior one, "it the number of tasks the service is running ..." vs this one "it adds new ec2 resources"  Minor nit, though.